.binstar.yml

```diff
@@ -1,15 +1,13 @@
 package: poliastro
 install:
-  - conda config --set always_yes true
+  - conda config --set always_yes yes --set changeps1 no
   - conda config --add channels poliastro
 script:
-  # Force conda build to use specific versions of Python and Numpy
-  - export CONDA_PY=$(python -c "from __future__ import print_function;from sys import version_info as ver;print(str(ver.major) + str(ver.minor))")
-  - export CONDA_NPY=$(python -c "from __future__ import print_function;import numpy;npver=numpy.__version__;print(npver[:npver.rfind('.')].replace('.',''))")
-  # Build once the env variables are set
-  - conda build buildscripts/condarecipe
```

Comment:
Yes, calling it a hell is an accurate description... Let's hope we'll have this sorted sometime soon, we'll have to contact Anaconda devs eventually.